stable/netsil/templates/service.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "fullname" . }}-stream-processor
+  labels:
+    app: {{ template "name" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    component: stream-processor
+spec:
+  selector:
+    app: {{ template "name" . }}
+    release: {{ .Release.Name }}
+    component: stream-processor
+  ports:
+    - name: http
+      port: 80
+    - name: https
+      port: 443
+      nodePort: 30443
+    - name: port2000
+      port: 2000
+    - name: port2001
+      port: 2001
+    - name: port2003
+      port: 2003
+    - name: port2003udp
+      port: 2003
+      protocol: UDP
+  type: NodePort
```

Comment:
Can you make the service type configurable?

Reply:
Is this a requirement or a nice to have?

Reply:
done.